src/plugins/intel_npu/src/plugin/npuw/compiled_model.cpp

```diff
@@ -112,6 +112,10 @@ ov::npuw::CompiledModel::CompiledModel(const std::shared_ptr<ov::Model>& model,
         LOG_INFO("Accuracy check is enabled.");
     }
 
+    // Initialize weights bank
+    const std::string weights_bank_opt = m_cfg.get<::intel_npu::NPUW_WEIGHTS_BANK>();
```

Comment:
so the default value is empty string, is that's right?

Reply:
Yes. Then in case of empty string I'll get a non-shared bank